algorithms/linfa-elasticnet/src/algorithm.rs

```diff
@@ -75,6 +152,28 @@ impl<F: Float, D: Data<Elem = F>> PredictInplace<ArrayBase<D, Ix2>, Array1<F>> f
     }
 }
 
+impl<F: Float, D: Data<Elem = F>> PredictInplace<ArrayBase<D, Ix2>, Array2<F>>
+    for MultiTaskElasticNet<F>
+{
+    /// Given an input matrix `X`, with shape `(n_samples, n_features)`,
+    /// `predict` returns the target variable according to elastic net
+    /// learned from the training data distribution.
+    fn predict_inplace(&self, x: &ArrayBase<D, Ix2>, y: &mut Array2<F>) {
+        assert_eq!(
+            x.nrows(),
+            y.nrows(),
+            "The number of data points must match the number of output targets."
+        );
+
+        general_mat_mul(F::one(), &x, &self.hyperplane, F::one(), y);
+        // *y = *y + self.intercept;
+    }
+
+    fn default_target(&self, x: &ArrayBase<D, Ix2>) -> Array2<F> {
+        Array2::zeros((x.nrows(), x.nrows()))
```

Comment:
Shouldn't it be `(x.nrows(), x.ncols())`

Reply:
It should actually be `(x.nrows(), y.ncols())`